test/integration/keyboard.integration.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable @typescript-eslint/no-loop-func */
+import { test, expect, Page, Locator } from '@playwright/test';
+
+import senseHorizon from '../rendering/theme/sense-horizon';
```

Comment:
Don't have to do it in this PR but shared code like this should be move to a folder that is not under rendering or integration tests. Otherwise it can be easy to think that a change in this file only affect the rendering tests, when it affects integration tests as well.

Reply:
Nice catch, okay, I'm going to pay attention at this.